packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -468,24 +466,24 @@ export class SequentialHandler implements IHandler {
 			}
 			// Since this is not a server side issue, the next request should pass, so we don't bump the retries counter
 			return this.runRequest(routeId, url, options, requestData, retries);
-		} else if (res.status >= 500 && res.status < 600) {
+		} else if (status >= 500 && status < 600) {
 			// Retry the specified number of times for possible server side issues
 			if (retries !== this.manager.options.retries) {
 				return this.runRequest(routeId, url, options, requestData, ++retries);
 			}
 			// We are out of retries, throw an error
-			throw new HTTPError(res.statusText, res.constructor.name, res.status, method, url, requestData);
+			throw new HTTPError(status.toString(), res.constructor.name, status, method, url, requestData);
```

Comment:
statusText is now redundant so maybe we should remove it if undici doesn't return anything similar